### PR TITLE
Using prettier to format code with GitHub-action

### LINKED ISSUE #35: Make a Github action bot that automatically does the file formatting.
**Is your feature request related to a problem? Please describe.**

By using [Github Action](https://help.github.com/en/actions) & [Prettier](https://prettier.io/), make a workflow that is automatically triggered when particular file formats (which are supported by prettier like .HTML,.css.js, etc)  are altered in the PR. For reference, you can go through [this article ](https://stefanzweifel.io/posts/run-prettier-or-php-cs-fixer-with-github-actions/).

**Describe the solution you'd like**
A GH action that automatically applies prettier on all HTML, CSS & JS files.

### PR DESCRIPTION
**Pull request by tushar5526**
Closes #35 
This PR adds a workflow which uses prettier to auto format the files with `js css vue jsx` files 
Steps Followed :
- Initialized npm in the repo
- Added .gitignore to ignore node-modules
- Added the workflow in prettier.yml inside .github/workflows

### CODE REVIEW

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+name: Format (prettier)
+
+on:
+    pull_request:
+        paths:
```
Comment (und3fined-v01d) [inline .github/workflows/prettier.yml:6]: Please include `jsx` as well

File: package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/tushar5526/simple-static-website/issues"
+  },
+  "homepage": "https://github.com/tushar5526/simple-static-website#readme",
+  "devDependencies": {
+    "prettier": "^2.0.5"
```
Comment (und3fined-v01d) [inline package.json:21]: Remove the ^, let us do this to ensure the peer dependency updates don't lead to security issues. I notice you have used yarn,

In that case just re-run

`yarn add -E prettier`

File: .github/workflows/prettier.yml (hunk 1 above)
Comment (tushar5526) [reply .github/workflows/prettier.yml:6]: Is it `.JSX` or `.jsx` ?

**Comment by tushar5526** [comment]
Done @und3fined-v01d

**Comment by rahul799** [comment]
Good work @tushar5526 . Now I want that this workflow must trigger whenever someone comment's "/style" (without inverted commas), in the PR discussion not like the current version which is triggered on each & every PR.

**Comment by tushar5526** [comment]
I will open a new PR for this, ran yarn on each file by mistake and pushed it :P

**Pull request closed without merging**